dht/token.cc

```diff
@@ -53,6 +55,14 @@ maximum_token() noexcept {
     return max_token;
 }
 
+token smallest_token() noexcept {
+    return smallest_token_static;
+}
+
+token greatest_token() noexcept {
+    return greatest_token_static;
+}
+
```

Comment:
token is so tiny, these could be inlines. I don't mind doing this later.

Reply:
(and no need for static objects, they just pessimize the code, construct everything inline)

Reply:
Right.